### ISSUE TITLE
Cant run examples

### ISSUE THREAD

**Issue by Wandalen** [RustAudio/cpal#610]
I tried to run example and got compile-time errors.

```
cargo run --example beep 
```

```
error[E0308]: mismatched types                                                                                                                                                                                                          
   --> src/host/alsa/mod.rs:245:26                                                                                                                                                                                                      
    |                                                                                                                                                                                                                                   
244 |         let handle = match handle_result {                                                                                                                                                                                        
    |                            ------------- this expression has type `std::result::Result<host::alsa::alsa::PCM, (host::alsa::alsa::Error, Option<host::alsa::alsa::nix::errno::Errno>)>`                                            
245 |             Err((_, Some(nix::errno::Errno::EBUSY))) => {                                                                                                                                                                         
    |                          ^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `host::alsa::alsa::nix::errno::Errno`, found enum `nix::errno::Errno`                                                                                             
    |                                                                                                                                                                                                                                   
    = note: perhaps two different versions of crate `nix` are being used?                                                                                                                                                               
                                                                                                                                                                                                                                        
error[E0308]: mismatched types                                                                                                                                                                                                          
   --> src/host/alsa/mod.rs:248:26                                                                                                                                                                                                      
    |                                                                                                                                                                                                                                   
244 |         let handle = match handle_result {                                                                                                                                                                                        
    |                            ------------- this expression has type `std::result::Result<host::alsa::alsa::PCM, (host::alsa::alsa::Error, Option<host::alsa::alsa::nix::errno::Errno>)>`                                            
...                                                                                                                                                                                                                                     
248 |             Err((_, Some(nix::errno::Errno::EINVAL))) => {
    |                          ^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `host::alsa::alsa::nix::errno::Errno`, found enum `nix::errno::Errno`
    |
    = note: perhaps two different versions of crate `nix` are being used?

```

Did I miss something?

**Comment by est31** [comment]
Have you tried latest master with #609 applied?

**Comment by kos-my** [comment]
I ran master 11 hours ago( fresh clone ). So 609 was applied.

**Comment by Wandalen** [comment]
Hi! I ran master 11 hours ago( fresh clone ). So 609 was applied.

**Comment by est31** [comment]
Hmm I wonder how even the CI of #609 managed to be green. Anyways, I've made a PR to fix this issue: #611
